### ISSUE TITLE
A command to leave sidebar without opening a file

### ISSUE THREAD

**Issue by finalpatch** [jojojames/dired-sidebar#12]
It would be nice to have a command that let the user go back to the last used editing window without opening a file or closing the sidebar.

In my current setup with neotree,  I have a function does this:

if (current window is sidebar)
    switch to most recent editing window
else
    switch to sidebar (open it if it's not already open)

neotree doesn't have a function for this either,  but it provides a (get-mru-window) function which makes it very easy to put together something for myself.

**Comment by jojojames** [comment]
Sounds reasonable. I guess this is contingent on https://github.com/jojojames/dired-sidebar/issues/11.

**Comment by jojojames** [comment]
Since get-mru-window is built in, you should be able to write this function yourself.

I'm open to PRs if you want to get that command pushed upstream. Thanks! :)
